src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -202,7 +202,7 @@ public static function phpize($value)
                 return '0x' === $value[0].$value[1] ? hexdec($value) : (float) $value;
             case preg_match('/^0x[0-9a-f]++$/i', $value):
                 return hexdec($value);
-            case preg_match('/^(-|\+)?[0-9]+(\.[0-9]+)?$/', $value):
+            case preg_match('/^(-|\+)?\d+(\.\d+)?$/', $value):
```

Comment:
Not really,

I didn't introduce /u modifier, so `[0-9] -> \d` not changing behavior,
but can be safely switched to use unicode later (if the feature requested just /u modifier needs to be added).